docs/sphinx/usage/Interpreting-Results.rst

```diff
@@ -25,18 +27,12 @@ definition of the return values is listed below:
      - A known **vulnerability** has been detected
    * - WARNING
      - We have detected something that could be a vulnerability but **manual analysis is required** to confirm (inconclusive)
-   * - SKIPPED NOT IMPLEMENTED
```

Comment:
Are we not using "Not Implemented" anymore?

Reply:
Don't think we ever used it.